apps/src/dance/Dance.js

```diff
@@ -124,15 +125,18 @@ Dance.prototype.init = async function (config) {
 
   ReactDOM.render((
     <Provider store={getStore()}>
-      <AppView
-        visualizationColumn={
-          <DanceVisualizationColumn
-            showFinishButton={showFinishButton}
-            retrieveMetadata={this.updateSongMetadata.bind(this)}
-          />
-        }
-        onMount={onMount}
-      />
+      <div>
+        <SignInOrAgeDialog danceStyle={true}/>
```

Comment:
You lost me with what `danceStyle` means here?

Reply:
Ohh - whoa - I was way off.  `danceStyle` to me meant things like rap, salsa, etc 😂.  What if this prop was `useDancePartyStyles`? That's more obvious to me to have a true/false value.

Reply:
:joy: I see what you mean! Yup - use DancePartyStyles sounds much better.